docs/language/contract-updatability.md

```diff
@@ -399,3 +399,36 @@ Similar to functions, constructors are also not stored. Hence, any changes to co
 A contract may import declarations (types, functions, variables, etc.) from other programs. These imported programs are
 already validated at the time of their deployment. Hence, there is no need for validating any declaration every time
 they are imported.
+
+# Cadence 1.0 Contract Updates
```

Comment:
Maybe this should also be referenced in the [migration guide](https://github.com/onflow/cadence-lang.org/tree/main/docs/cadence_migration_guide)